plugins/pulp_rpm/plugins/migrations/0025_export_relative_url.py

```diff
@@ -0,0 +1,66 @@
+"""
+Migration to normalize relative_url, and populate relative_url on
+export distributors
+"""
+from pulp.server.db import connection
+
+
+def _find_yum_distributor(distributors):
+    """
+    Find this repo's 'yum_distributor' to pull the relative_url from.
+
+    While technically you can have more than one yum_distributor...
+    functionality really only works for a single distributor at a time.
+    This distributor's ID is 'yum_distributor'
+
+    :param distributors: List of distributors for repo.
+    """
+    for distributor in distributors:
+        if distributor['distributor_id'] == 'yum_distributor':
+            if 'config' in distributor and 'relative_url' in distributor['config']:
+                return distributor
+    return None
+
+
+def _clean_distributors_relative_url(repo_distributors, distributors):
+    for distributor in distributors:
+        if 'config' in distributor and 'relative_url' in distributor['config']:
+            # Yum distributor's relative_url was being generated with a
+            # leading slash before PR #776, which fixes issue #1520
+            relative_url = distributor['config']['relative_url'].lstrip('/ ')
+            distributor['config']['relative_url'] = relative_url
+            repo_distributors.update({'_id': distributor['_id']},
+                                     {'$set': {'config': distributor['config']}})
```

Comment:
I am intentionally only updating this specific distributor with it's new value with the match on it's object id, '_id'